src/components/shared/node_count_selector.js

```diff
@@ -146,10 +161,10 @@ class NodeCountSelector extends React.Component {
             >
               <NumberPicker
                 label=''
-                max={999}
-                min={1}
+                min={valueConstraints.min}
+                max={valueConstraints.max}
                 onChange={this.updateNodeCount}
-                readOnly={false} // TODO
+                readOnly={readOnly} // TODO
```

Comment:
Why is this TODO?

Reply:
Was there before, not sure what it means, since there's no additional info

Reply:
Then I think you can remove it. It will only raise questions again otherwise.